tensorflow/core/util/gpu_launch_config.h

```diff
@@ -117,6 +122,7 @@ struct GpuLaunchConfig {
   // Number of blocks for Gpu kernel launch.
   int block_count = -1;
 };
+using CudaLaunchConfig = GpuLaunchConfig;
```

Comment:
again within `#ifdef GOOGLE_CUDA`

was wondering having a `CREATE_CUDA_TYPE_ALIAS` macro would make things more consistent / digestible. At the very least it will prevent you from having to type `#ifdef GOOGLE_CUDA` everywhere

Reply:
I like the macro approach, will add the macro as you suggested.